templates/credentialling/self-study.html

```diff
@@ -0,0 +1,368 @@
+              Toggle side navigation
+            </a>
+          </div>
+      
+          <h3 class="p-side-navigation__heading">Self Study Resources</h3>
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#general-resources">General Resources</a>
+            </li>
+          </ul>
+          <h3 class="p-side-navigation__heading">Linux Essentials resources</h3>
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#le-tutorials">Tutorials</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#le-documentation">Documentation</a>
+            </li>
+          </ul>
+          <h3 class="p-side-navigation__heading">Desktop Essentials resources</h3>
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#de-tutorials">Tutorials</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#de-documentation">Documentation</a>
+            </li>
+          </ul>
+          <h3 class="p-side-navigation__heading">Server Essentials resources</h3>
+          <ul class="p-side-navigation__list">
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#se-tutorials">Tutorials</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#se-documentation">Documentation</a>
+            </li>
+            <li class="p-side-navigation__item">
+              <a class="p-side-navigation__link" href="#se-webinar">Webinars</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+    <div class="col-9">
+      <h1>Self-study resources</h1>
+      <p>The links on this page lead to documentation and tutorials about topics relevant to Canonical certification.</p>
+      <p>Although they do not constitute an exhaustive list of the subjects covered by Canonical certifications, they provide a good starting point for your studies. Use them to familiarise yourself with key concepts, or to refresh your existing knowledge.</p>
+      <h2 id="general-resources">General resources</h2>
+      <p>There are a number of instructional resources available, both community- and Canonical-made, that contain a wealth of useful information. The following links lead to Canonical’s official educational resources.</p>
+      <ul>
+        <li>
+          <a href="https://ubuntu.com/tutorials">Tutorials</a>
+        </li>
+        <li>
+          <a href="https://help.ubuntu.com/">Ubuntu documentation</a>
+        </li>
+        <li>
+          <ul>
```

Comment:
Something is going wrong here. We introduce two bullets in front of the Desktop entry somehow. We need to eliminate the first, and have only the nested bullets.

Once that's resolved, this is perfect.

Reply:
![Screenshot from 2022-09-30 13-11-32](https://user-images.githubusercontent.com/64434968/193322176-bb52e107-72a2-4f52-b8fd-882fa0cf89d1.png)

Reply:
Good catch, I've made the fix.